tensorflow/stream_executor/rocm/rocm_dnn.cc

```diff
@@ -96,6 +98,24 @@ string ToString(miopenStatus_t status) {
   }
 }
 
+string ToString(miopenConvAlgorithm_t algorithm) {
```

Comment:
Is it possible to use [this header](https://github.com/AMDComputeLibraries/MLOpen/blob/107449a8ed1d0afd988c9da4e94e430cbc0afd02/src/include/miopen/conv_algo_name.hpp#L37); together with [this implementation](https://github.com/AMDComputeLibraries/MLOpen/blob/107449a8ed1d0afd988c9da4e94e430cbc0afd02/src/conv_algo_name.cpp#L68) So that we are consistent with actual MIOpen algorithm names?

Reply:
will look into it. I had rather not include any header file other than `miopen.h`, but that is not a strong preference